src/app/pages/home/components/home-tabs.tsx

```diff
@@ -13,53 +14,54 @@ interface HomeTabsProps extends StackProps {
   activity: React.JSX.Element;
 }
 
-const ANALYTICS_PATH = ['/balances', '/activity'];
+export function HomeTabs({ balances, activity }: HomeTabsProps) {
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
 
-export function HomeTabs(props: HomeTabsProps) {
-  const { balances, activity, ...rest } = props;
-  const analytics = useAnalytics();
+  const tabs = useMemo(
+    () => [
+      { slug: RouteUrls.Home, label: 'Balances' },
+      { slug: RouteUrls.Activities, label: 'Activity' },
+    ],
+    []
+  );
 
-  const { activeTab, setActiveTab } = useHomeTabs();
+  const getActiveTab = useCallback(
+    () => tabs.findIndex(tab => tab.slug === pathname),
+    [tabs, pathname]
+  );
 
```

Comment:
I'm not sure if we still want to trigger analytics here? I can add it. 

Now we will track a page view on the `/activities` route and can infer balances are seen at `/`

Reply:
Nope, this can be killed now that we're using routes